rts/lra/lra-coordinator-jar/pom.xml

```diff
@@ -186,9 +186,6 @@
                                 <lra-participant.port>9080</lra-participant.port>
                                 <server.jvm.args>${server.jvm.args}</server.jvm.args>
                             </systemPropertyVariables>
-                            <excludes>
-                              <exclude>AnnotationsTestCase.java</exclude>
```

Comment:
@mmusgrov is this tidy up? If so can it be added to a separate commit (same PR is fine)

Reply:
I will delete the tidy up

Reply:
@tomjenkinson Try that latest push instead. The PR has passed 4 times (zero failures) so I think this fix looks positive.